arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -4534,7 +4559,7 @@
 					"@executable_path/Frameworks",
 				);
 				OTHER_LDFLAGS = "";
-				PRODUCT_BUNDLE_IDENTIFIER = "com.esri.arcgis-ios-sdk-samples";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.esri.arcgis-ios-sdk-samplesV";
```

Comment:
why is the bundle identifier name changed?

Reply:
Divesh and I had trouble running this app on a physical device. This was his solution that allows me to test the sample using a physical device.

Reply:
@vquach2404 The reason you had to change the bundle identifier is because you're using the enterprise team. For deploying this app to device, you'll want to use the non-enterprise team. In addition, please do not check in changes like this to the bundle identifier and/or development team.

Reply:
Can someone give me access to the non-enterprise team? I don't think I have access, it's not listed in the drop down menu under teams.